### DISCUSSION TITLE
Organizing Contest within the community

### DISCUSSION THREAD

**Discussion by octonawish-akcodes** [EddieHubCommunity/support#3859] [Ideas]
We should organize some tech events like quiz championships, designing contest kind of contests where the server is divided into some teams have to achieve their goal as faster as they can in order to benefit their overall team which can ultimately help all the members to learn and how to apply things in real time in a fun manner.
And having prizes for all the participant team and a surprise prize to the winning team.

**Comment by jaynarayan-vaishnav** [comment]
So basically you are saying that we should start small scale hackathons in the Eddiehub community??

**Comment by eddiejaoude** [comment]
Thanks for the suggestion. We are trying to plan some events 👍  

Having prizes for "all participants" requires a lot of admin work and also a lot of money, for example when sending a few stickers costs about $5 per person

We will probably only be able to do give aways to a limit amount of people
